scrapy/utils/reactor.py

```diff
@@ -90,6 +90,24 @@ def verify_installed_reactor(reactor_path):
         raise Exception(msg)
 
 
+def verify_installed_loop(loop_path):
+    from twisted.internet import reactor
+    loop_class = load_object(loop_path)
+    if isinstance(reactor._asyncioEventloop, loop_class):
+        return
+    installed = (
+        f"{reactor._asyncioEventloop.__class__.__module__}"
+        f".{reactor._asyncioEventloop.__class__.__qualname__}"
+    )
+    specified = f"{loop_class.__module__}.{loop_class.__qualname__}"
+    raise Exception(
+        f"Scrapy found an asyncio Twisted reactor already "
+        f"installed, and its event loop class ({installed}) does "
+        f"not match the one specified in the ASYNCIO_EVENT_LOOP "
```

Comment:
```suggestion
        "Scrapy found an asyncio Twisted reactor already "
        f"installed, and its event loop class ({installed}) does "
        "not match the one specified in the ASYNCIO_EVENT_LOOP "
```
Nitpicking. 🤓

Reply:
Cool! I was not aware that this produce the same byte code, basically this is cosmetic preference then. 💅🏻 I tend to use this way because is explicit that you have a string interporlation in that line.